File: delta3d/examples/testPython/flyplane.py
from Delta import *
from math import *
from time import *

plane = Object.GetInstance('cessna')

transform = Transform()

angle = 0

def radians(v):
   return v * pi/180

while True:
   transform.Set(40*cos(radians(angle)),
                 100 + 40*sin(radians(angle)), 
                 0, angle, 0, -45)
   plane.SetTransform(transform)
   sleep(0.01)
   angle -= 0.45

File: tags/VER0_3_0/delta3d/examples/testLua/flyplane.lua
plane = P51.GetObjectInstance('cessna')

transform = P51.Transform()

angle = 0

t = os.clock()

while true do
   transform:SetTranslation(40*math.cos(math.rad(angle)),
                            100 + 40*math.sin(math.rad(angle)), 
                            0)
   transform:SetRotation(angle, 0, -45)
   plane:SetTransform(transform)
   nt = os.clock()
   angle = angle - 45*(nt-t)
   t = nt
end